src/useStorage/README.md

```diff
@@ -0,0 +1,45 @@
+# useStorage
```

Comment:
I just want to verify, if someone were to manually change a value in storage with `localStorage.set()`, this composition's value ref would NOT know about it, correct? 

Perhaps we should make sure that's noted in the readme?

Reply:
Correct, but that is also expected. There is no browser api for watching/listening to changes to storage. If you don't go through the composition there shouldn't be any expectation that your update will be reactive.